Add more porting providers to Console support in Godot (#5267)
godotengine/godot-docs: tutorials/platform/consoles.rst

Following is the list of providers:

- `Deck13 Interactive <https://www.deck13.com/>`_ offers Switch porting
  and publishing of Godot games.

> - `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
>   Switch and PS4 porting and publishing of Godot games.

Reviewer: ```suggestion
- `Lone Wolf Technology <http://www.lonewolftechnology.com/>`_ offers
  Switch and PlayStation 4 porting and publishing of Godot games.
```

- `Pineapple Works <https://pineapple.works/>`_ offers
  Switch and Xbox One porting and publishing of Godot games.

> - `Ratalaika Games <https://www.ratalaikagames.com//>`_ offers

Reviewer: ```suggestion
- `Ratalaika Games <https://www.ratalaikagames.com/>`_ offers
```

> - `Ratalaika Games <https://www.ratalaikagames.com//>`_ offers
>   Switch, Xbox One, XBox Series X, Playstation 4, and Playstation 5 porting
>   and publishing of Godot games.

Reviewer: ```suggestion
- `Ratalaika Games <https://www.ratalaikagames.com//>`_ offers
  Switch, Xbox One, Xbox Series, PlayStation 4, and PlayStation 5 porting
  and publishing of Godot games.
```
